settings.gradle

```diff
@@ -1,3 +1,50 @@
+pluginManagement {
+    gradle.ext.kotlinVersion = '1.6.10'
+    gradle.ext.agpVersion = '4.2.2'
```

Comment:
Warning (⚠️): @oguzkocer FYI and due to the merge conflicts after updating `trunk`, this dependency wasn't updated to `7.1.1`.

Reply:
Nice catch! Fixed in d573663e44a753bd7d06ead2f4be9421f3b41d8a.